internal/pkg/aws/cloudformation/cloudformation.go

```diff
@@ -218,6 +218,19 @@ func (c *CloudFormation) TemplateBodyFromChangeSet(changeSetID, stackName string
 	return aws.StringValue(out.TemplateBody), nil
 }
 
+// Outputs returns the outputs of a stack description.
+func (c *CloudFormation) Outputs(stack *Stack) (map[string]string, error) {
+	stackDescription, err := c.Describe(stack.Name)
+	if err != nil {
```

Comment:
Will this ever return a "Stack does not exist" error? Should we handle it silently here rather than making higher level packages do error checking?

Reply:
Oh, good question, but I'm not sure I understand entirely. ErrStackNotFound is already being returned by Describe(), and Outputs() is  wrapping it. What do you mean by "handle it silently"?

Reply:
Return an empty map and nil instead of an error if the stack is not found. It depends on how we expect higher level packages to do error checking and handling--like if we generally expect higher level packages to check for the lower level `ErrStackNotFound` vs gracefully handling a zero length output and only returning an error for unexpected lower level errors.

Reply:
I think I'll leave as is for now because I think I'd want to know that `ErrStackNotFound` was the cause of whatever higher level action to act unexpectedly.